src/Components/Map.js

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useState, useRef } from "react";
-import mapboxgl from '!mapbox-gl'; // eslint-disable-line import/no-webpack-loader-syntax 
+import mapboxgl from 'mapbox-gl'; // eslint-disable-line import/no-webpack-loader-syntax 
```

Comment:
You shouldn't remove the `!` as this will break the map

the `!` is used to exclude the map from the being transpilied while building the project

Reply:
got it, I will take a look

Reply:
Have added back the `!` syntax in a commit below